README.md

```diff
@@ -0,0 +1,48 @@
+# CODE CHALLENGE
```

Comment:
Great README super clear and included everything to get the app running on my machine